assets/cluster-monitoring-operator/prometheus-rule.yaml

```diff
@@ -28,6 +28,37 @@ spec:
       for: 15m
       labels:
         severity: warning
+    - alert: HighlyAvailableWorkloadIncorrectlySpread
+      annotations:
+        description: Workload {{ $labels.namespace }}/{{ $labels.workload }} is incorrectly
+          spread across mutliple nodes which breaks high-availability requirements.
+          There are {{ $value }} pods on node {{ $labels.node }}, where there should
+          only be one. Since the workload is using persistent volumesm it is not fixable
+          by operators, but you can follow the guidelines provided in the runbook
```

Comment:
```suggestion
          only be one. Since the workload is using persistent volumes manual intervention is needed.
           Please follow the guidelines provided in the runbook
```